src/views/Home.jsx

```diff
@@ -14,6 +25,30 @@ export function Home({ makeNewList, handleError }) {
 			</p>
 			{handleError ? <span>{handleError}</span> : null}
 			<button onClick={handleClick}>Create a new list!</button>
+
+			<div className="JoinListForm">
+				<p>Join an existing shopping list by entering a three word token.</p>
+				{joinListErrorMsg ? <span>{joinListErrorMsg}</span> : null}
+				<form onSubmit={handleJoinList}>
+					<label htmlFor="input">Share token</label>
```

Comment:
I also have a small aesthetic suggestion to perhaps change "Share token" to "Enter token" for more clarity. I initially misunderstood and thought this was meant for sharing a token with another user.

Reply:
I do prefer your suggestion for "Enter token" over the "Share token" that is demonstrated in our wireframe from issue #3